proto/iotics/api/common.proto

```diff
@@ -175,12 +169,18 @@ message TwinID {
   string value = 1;
 }
 
-// FeedID is a unique feed identifier (scoped to the TwinID).
+// FeedID is a unique feed identifier (scoped to the set of feed for a TwinID).
 message FeedID {
```

Comment:
yep, I agree about that. We want to get rid of the wrapper types but this would be a breaking change for all the services. We will keep that for later.